_posts/newsletters/2022-01-31-newsletter.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title: US-RSE Newsletter
+subtitle: January 2022 Newsletter
+category: newsletter
+tags: [newsletter, survey, sc22, scipy22]
+---
+
+In this monthly newsletter, we share recent, current, and planned activities of the US-RSE Association, and related news that we think is of interest to US-RSE members. Newsletters are also available on our [website](https://us-rse.org/newsletters/) alongside the growing resources and information on the US-RSE Association. To receive our newsletter, join US-RSE [here](https://us-rse.org/join/).
+
+In this issue:
+
+* [International RSE Survey](#survey)
+* [US-RSE Community Workshop](#workshop)
+* [US-RSE Community Calls](#community-calls)
+* [RSE Stories Podcast](#podcast)
+* [US-RSE at SciPy 2022](#scipy)
+* [US-RSE at SC22](#sc22)
+* [Recent Job Postings](#jobs)
+* [Community News](#news)
+* [DEI Media Club Discussion](#dei-mc)
+* [Interesting Events](#events)
+* [Interesting Reads](#reads)
+* [US-RSE Outreach](#outreach)
+* [US-RSE Financials](#financials)
+* [2022 Steering Committee Roles](#steering-committee)
+* [Get Involved](#involved)
+
+<a name="survey"></a>
+# **International RSE Survey**
+
+The [2021 RSE Survey](https://softwaresaved.limequery.com/386272?lang=en) is still open. You don’t need to be called an RSE to complete the survey. Your formal job title does not matter: we want to hear from you if you develop software for academic research as part of your work. The survey provides considerable detail about the RSE role: a person’s route into the role, the type of work they conduct, their working practices, job satisfaction and demographics. This means it takes around 20 minutes to complete. We appreciate that this is a significant amount of your time to donate - and we thank you for it!
```

Comment:
What about RSE allys? This sounds like "just research software engineer" types as opposed to anyone else. Is that the case?

Reply:
Yes. It's really intended for just RSEs. The questions aren't going to be well suited for those not in an RSE role. Even as a manager of RSEs, a number of questions are not applicable to me anymore.